docs/elements.md

```diff
@@ -190,6 +190,47 @@ Two grading methods are available when using `partial-credit="true"`:
 
 -----
 
+## `pl-dropdown` element
```

Comment:
You need to include the new element in the table of contents

Reply:
Done.